doc/howto/storage_list_pools.md

```diff
@@ -0,0 +1,15 @@
+# How to list storage pools
+
+You can display a list of all available storage pools and check their configuration.
+
+## Show storage pool information
```

Comment:
No need for this subheading, unless we want to make it really clear that we're documenting both "list" and "show" - in that case, we need a second heading before line 13.

I don't really think that's necessary though.